oeps/oep-0045-arch-ops-and-config.rst

```diff
@@ -0,0 +1,175 @@
+##########################################
+OEP-45: Configuring and Operating Open edX 
+##########################################
+
+.. list-table::
+
+   * - OEP
+     - :doc:`OEP-0045 </oeps/oep-0045>`
+   * - Title
+     - Configuring and Operating Open edX
+   * - Last Modified
+     - 2020-04-06
+   * - Authors
+     - Bill DeRusha
+   * - Arbiter
+     - Felipe Montoya 
+   * - Status
+     - Under Review
+   * - Type
+     - Architecture
+   * - Created
+     - 2020-04-06
+   * - Review Period
+     - 2020-04-17 - 2020-05-04
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Abstract
+========
+
+Wherever possible, The edX organization will provide and manage Docker images as the medium for packaging operating dependencies of Open edX applications rather than providing Ansible playbooks.
+
+The configuration for running applications in different environments will be done through a config file per unique [application, environment] combination. How those files are generated and managed is the responsibility of each operator.
+
+All standard operations for running and managing any Open edX application will be documented. How those procedures are executed is the responsibility of each operator.
+
+Wherever possible, operating concerns will live within the codebase they are concerned with. Thus it should be expected that all applications will provide the following:
+
+* A Dockerfile from which the container image can be built.
+* A documented settings file with production-ready defaults.
+* A documented settings file with all of the configuration values that must be defined by the operator.
```

Comment:
This is referring to the `required.py` module, right?  _(May want to just add the file's names here and above for added clarity.)_

Can we clarify this as:
A documented settings file with **remaining (or other)** configuration values that must be defined by the operator.